apps/web/src/design-system/button/Button.styles.ts

```diff
@@ -8,17 +8,19 @@ const getLabelStyles = () => ({
   backgroundClip: 'text',
   fontWeight: 'bold',
 });
+
 const getFilledDisabledStyles = (theme) => ({
   backgroundImage: theme.colorScheme === 'dark' ? getGradient(colors.B20) : getGradient(colors.B98),
 });
+
 const getFilledStyles = (theme) => ({
   border: 'transparent',
   boxShadow: theme.colorScheme === 'dark' ? shadows.dark : shadows.color,
 });
 
-const getOutlineStyles = (theme, disabled) => {
+const getOutlineStyles = (theme, disabled, variant?: 'outline' | 'filled') => {
```

Comment:
Why did you need the variant here? 🤔
I think it goes by class name of the variant